hudi-common/src/main/java/org/apache/hudi/common/fs/StorageSchemes.java

```diff
@@ -97,4 +104,12 @@ public static boolean isAppendSupported(String scheme) {
     }
     return Arrays.stream(StorageSchemes.values()).anyMatch(s -> s.supportsAppend() && s.scheme.equals(scheme));
   }
+
+  public static boolean isWriteTransactional(String scheme) {
+    if (!isSchemeSupported(scheme)) {
+      throw new IllegalArgumentException("Unsupported scheme :" + scheme);
+    }
+
+    return Arrays.stream(StorageSchemes.values()).anyMatch(s -> s.isWriteTransactional() && s.scheme.equals(scheme));
```

Comment:
If we know that scheme is supported we can just do `Scheme.valueOf`

Reply:
ENUM could be named differently than "scheme". and hence we are doing it this way. if its a single field ENUM, we could do that you are suggesting.